scripts/cluster-install/install.py

```diff
@@ -0,0 +1,293 @@
+    rabbit1 = _get_vm(instances_details, 'rabbitmq_1')
+    rabbit2 = _get_vm(instances_details, 'rabbitmq_2')
+    rabbit3 = _get_vm(instances_details, 'rabbitmq_3')
+    manager1 = _get_vm(instances_details, 'manager_1')
+    manager2 = _get_vm(instances_details, 'manager_2')
+    manager3 = _get_vm(instances_details, 'manager_3')
+    instances_list = [factory, db1, db2, db3, rabbit1, rabbit2, rabbit3,
+                      manager1, manager2, manager3]
+    if len(instances_details) > 10:
+        load_balancer = _get_vm(instances_details, 'load_balancer')
+        instances_list.append(load_balancer)
+    return instances_list
+
+
+def _install_load_balancer(instances_list, key_file_path):
+    logging.info('Installing load balancer')
+    load_balancer = instances_list[-1]
+    managers_details = ''
+    client = _create_ssh_client(load_balancer.public_ip, 'centos',
+                                key_file_path)
+    os.system('scp -i {0} -o "StrictHostKeyChecking no" -r {1} '
+              'centos@{2}:/home/centos'.format(key_file_path,
+                                               LOCAL_INSTALL_CLUSTER + '/certs',
+                                               load_balancer.public_ip))
+    os.system('scp -i {0} -o "StrictHostKeyChecking no" {1} '
+              'centos@{2}:/home/centos'.format(key_file_path,
+                                               'install_load_balancer.sh',
+                                               load_balancer.public_ip))
+    for manager in instances_list[7:10]:
+        managers_details += (manager.private_ip + ' ' +
+                             manager.public_ip + ' ')
+    install_command = 'sudo bash /home/centos/install_load_balancer.sh {}'.\
+        format(managers_details)
+    stdin, stdout, stderr = _blocking_exec_command(client, install_command)
+    print(stdout.read())
+    return
+
+
+def _create_install_cluster_directory(license_path):
+    logging.info('Creating `install_cluster` directory')
+    if os.path.exists(LOCAL_INSTALL_CLUSTER):
+        os.system('rm -r {}'.format(LOCAL_INSTALL_CLUSTER))
+    os.mkdir(LOCAL_INSTALL_CLUSTER)
+    os.mkdir(LOCAL_INSTALL_CLUSTER + '/certs')
+    _download_manager_and_license(license_path)
+
+
+def _parse_command():
+    parser = argparse.ArgumentParser(description='Installing an Active-Active '
+                                                 'manager cluster')
+    parser.add_argument('--config-path', action='store', type=str,
+                        required=True, help='The config.yaml file path')
+    parser.add_argument('--do-not-clean-on-failure', dest='clean',
+                        action='store_false', default=True,
+                        help='Pass this flag if you do not want to clean '
+                             'your Openstack enviornment on failure')
+    return parser.parse_args().config_path, parser.parse_args().clean
+
+
+def _delete_factory(connection, servers_ids_list):
```

Comment:
_delete_factory isn't a particularly descriptive name for this. Can you name it something more related to what it's doing/intended to be doing?

Reply:
Done.